Dockerfile.builder

```diff
@@ -37,5 +37,5 @@ RUN apt-get update && \
 # Install Go
 #
 RUN curl -sfL https://dl.google.com/go/go${GO_VERSION}.linux-${ARCH}.tar.gz | tar -xzC /usr/local && \
-    GO111MODULE=on go get github.com/gordonklaus/ineffassign@1003c8bd00dc2869cb5ca5282e6ce33834fed514 && \
+    curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.27.0 && \
```

Comment:
I guess we'll need to build a new `cilium-builder` image and bump the version in `Dockerfile`. /cc @aanm

Reply:
For some reason I missed this. I think we can remove this line altoher. The `ineffassign` is not used anywhere from the docker builder.

Reply:
Thanks, just removed changes in this file :)